db/migrate/20150710110707_add_missing_fields_and_indexes_to_cloud_subnets.rb

```diff
@@ -0,0 +1,13 @@
+class AddMissingFieldsAndIndexesToCloudSubnets < ActiveRecord::Migration
+  def change
+    add_column :cloud_subnets, :cloud_tenant_id,                :bigint
+    add_column :cloud_subnets, :dns_nameservers,                :string
+    add_column :cloud_subnets, :ipv6_router_advertisement_mode, :string
+    add_column :cloud_subnets, :ipv6_address_mode,              :string
+    add_column :cloud_subnets, :extra_attributes,               :text
```

Comment:
btw. dns_nameservers is a string because it's defined as 

dns_nameservers	plain	xsd:string	
DNS server.